src/titlebar.ts

```diff
@@ -115,16 +115,18 @@ export class Titlebar extends Themebar {
 		this.registerTheme(this._options.iconsTheme);
 	}
 
+	private blur = () => {
+		this.onDidChangeWindowFocus(false);
+		this.onBlur();
+	}
+
 	private registerListeners() {
 		this.currentWindow.on(EventType.FOCUS, () => {
 			this.onDidChangeWindowFocus(true);
 			this.onFocus();
 		});
 
-		this.currentWindow.on(EventType.BLUR, () => {
-			this.onDidChangeWindowFocus(false);
-			this.onBlur();
```

Comment:
These lines blur the window, this causes the color of the title bar to change. It's a likely solution, but I'd have to refocus the window.

Reply:
Ahh good point, I didn't realize the colors changed on blur. I've updated this to just pass `this.menu.blur` directly to bypass the style changes.

Reply:
Sorry Alex, I didn't see all of the changes you requested after I force pushed one of the changes to my fork. Feel make changes directly to this MR if you want. Or, if you want to re-add the requested changes, I'll push any updates in new commits. Sorry about that!

Reply:
invalid